config/v16/config-docker.json

```diff
@@ -57,6 +57,48 @@
         "ISO15118PnCEnabled": true,
         "ContractValidationOffline": true
     },
+    "CostAndPrice": {
```

Comment:
nit:
Our examples configs are pretty minimal so far, so I would suggest to set `CustomDisplayCostAndPrice` to false and remove all other keys.

I opened an issue to also provide a fully featured config as part of v16: https://github.com/EVerest/libocpp/issues/736

Reply:
ok! i also changed config.json.